gcloud/bigtable/happybase/__init__.py

```diff
@@ -16,4 +16,136 @@
 
 This package is intended to emulate the HappyBase library using
 Google Cloud Bigtable as the backing store.
+
+Differences in Public API
+-------------------------
+
+Some concepts from HBase/Thrift do not map directly to the Cloud
+Bigtable API. As a result, the following instance methods and functions
+could not be implemented:
+
+* :meth:`.Connection.enable_table` - no concept of enabled/disabled
+* :meth:`.Connection.disable_table` - no concept of enabled/disabled
+* :meth:`.Connection.is_table_enabled` - no concept of enabled/disabled
+* :meth:`.Connection.compact_table` - table storage is opaque to user
+* :func:`make_row() <gcloud.bigtable.happybase.table.make_row>` - helper
+  needed for Thrift library
+* :func:`make_ordered_row() <gcloud.bigtable.happybase.table.make_ordered_row>`
+  - helper needed for Thrift library
+* :meth:`Table.regions() <gcloud.bigtable.happybase.table.Table.regions>`
+  - tables in Cloud Bigtable do not expose internal storage details
+* :meth:`Table.counter_set() \
+      <gcloud.bigtable.happybase.table.Table.counter_set>` - method can't
+  be atomic, so we disable it
+* The ``__version__`` value for the HappyBase package is :data:`None`
```

Comment:
I don't care either way, but it would be nice to have some indication of which version of happybase this is based on.